docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
+==- 1 - Total number of frames
 
-#### QOL Improvements
+The total number of frames in the video stream.
 
-- **Use slow.pics friendly file naming, so you can drag all the images onto the site and have them automatically sorted, with comparison names intact.**
+==- 2 - Video stream length
 
-   In VS-Preview on the bottom bar select Misc, and set file name template to `{frame}_{index}_{Name}`.
+The playback length of the video stream.
 
-- **Jump through frames quickly**
+==- 3 - Video stream resolution
 
-   In VS-Preview on the bottom bar select Playback, directly right of the playback keys set the `1` value to `120`.
+The content resolution of the video stream.
 
-- **Take screenshots quicker by setting the save image button to enter**
+==- 4 - Video format
 
-   Open `%localappdata%\Programs\Python\Python311\Lib\site-packages\vspreview\toolbars\misc` and edit `toolbar.py`
-    - Line 166: Replace `QKeyCombination(Qt.Modifier.SHIFT, Qt.Key.Key_S).toCombined(), self.save_frame_as_button.click` with `(Qt.Key_Return), self.save_frame_as_button.click`.
-    - If you can't spam fast enough, in vs-preview click settings and set PNG compression to a lower level (higher value).
+The chroma subsampling type and bit depth.
 
-- **Swap binds to save your pinky finger, so you no longer have to hold shift all the time**
+==- 5 - Source frame rate
 
-    Open `%localappdata%\Programs\Python\Python311\Lib\site-packages\vspreview\toolbars\playback\` and edit `toolbar.py`
+The frame rate of the video stream, represented by a fraction and approximate decimal.
 
-    - Select lines 180-187, delete them, and paste the following, making sure the indentation of the lines match (using spaces).
+==- 6 - Picture type of current frame
 
-    ```python
-    self.main.add_shortcut(QKeyCombination(Qt.SHIFT, Qt.Key.Key_Left), self.seek_to_prev_button.click)
-    self.main.add_shortcut(QKeyCombination(Qt.SHIFT, Qt.Key.Key_Right), self.seek_to_next_button.click)
-    self.main.add_shortcut(Qt.Key.Key_Left, self.seek_n_frames_b_button.click)
-    self.main.add_shortcut(Qt.Key.Key_Right, self.seek_n_frames_f_button.click)
-    ```
+The picture type of the current frame.
 
-#### Post-processing
+==-
 
-The Bolt-action method is best paired with some scripts to run after you've saved all the frames. The following scripts are generated by chatgpt as I cannot code 👍
-Generally, I just run all 3 with a .bat file, or just the first 2 if I'm not uploading to ptpimg.
-==- Zero padding
+## Comparing
 
-Zero pads all frames in the current directory and removes some of the garbage from the name. Allows for automatic numerical sorting on slow.pics and automatic filling of comparison names based on comp.vpy
+### Tips
 
-```python
-import os
+- *[Label your sources clearly to the user](#2-frame-saving)*
```

Comment:
won't this make more sense to refer to [here](https://github.com/Snaacky/thewiki/pull/167/files#diff-157e9ccb1422c31419ef3c22b113777c9e37830a8289acb9ade6ea084d673074R121-R124) instead

Reply:
I'm unable to see the link you attached.

Reply:
https://ricky8k.github.io/thewiki/tutorials/comparison/#recommended-source-naming